src/dev/src/main.ts

```diff
@@ -11,6 +12,8 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
+import { runCssVarsPolyfill } from '@clr/base/src';
```

Comment:
Path nit, does `import { runCssVarsPolyfill } from '@clr/base';` not work? If not we can look into the path mapping stuff in a later pr.